tutorials/commands/flux-jobs.rst

```diff
@@ -0,0 +1,344 @@
+    JOBID RC
+    f2TRHV5mR -128
+    f2SLdmS4F 1
+    f2SLfFRLb 1
+    f2RYByw1h 0
+    f2RYAVwjM 0
+
+There are many additional fields that are available for output in ``flux-jobs``.  This tutorial will not go through them but you can find information on them via the :core:man1:`flux-jobs` manpage as well as ways to format the output in pretty ways.
+
+For must users, instead of formatting your own output, you may wish to use one of the additional "common" formats available in ``flux-jobs``.  They can be listed with ``flux jobs --format=help``.
+
+.. code-block:: console
+
+    $ flux jobs --format=help
+
+    Configured flux-jobs output formats:
+
+      default      Default flux-jobs format string
+      cute         Cute flux-jobs format string (default with emojis)
+      long         Extended flux-jobs format string
+      deps         Show job urgency, priority, and dependencies
+
+For example, lets take a look at the ``long`` output.
+
+.. code-block:: console
+
+    $ flux jobs --format=long
+           JOBID USER     NAME          STATUS NTASKS NNODES     T_SUBMIT  T_REMAINING     TIME INFO
+       f2TRD37vP achu     sleep          SCHED      1      -  Mar29 17:28            -        - 
+       f2TREX7Cj achu     sleep          SCHED      1      -  Mar29 17:28            -        - 
+       f2TRG16V5 achu     sleep          SCHED      1      -  Mar29 17:28            -        - 
+       f2TRBZ8e3 achu     sleep            RUN      1      1  Mar29 17:28            -   53.01m catalyst160
+       f2TR8bA5N achu     sleep            RUN      1      1  Mar29 17:28            -   53.01m catalyst160
+       f2TR8bA5M achu     sleep            RUN      1      1  Mar29 17:28            -   53.01m catalyst160
+       f2TR77Ao1 achu     sleep            RUN      1      1  Mar29 17:28            -   53.01m catalyst160
+
+Compared to the default output, we have ``STATUS`` being output with full names instead of abbreviations, we have the time
+that the job was submitted via ``T_SUBMIT`` and the time remaining for the job in ``T_REMAINING`` (in this particular example, there is no time limit, thus no time remaining listed).
+
+.. note::
+
+    You can set the default output of ``flux jobs`` through the
+    environment variable FLUX_JOBS_FORMAT_DEFAULT.  For example, by
+    setting FLUX_JOBS_FORMAT_DEFAULT=long, the long output will be
+    output as the default output.
+
+.. note::
+
+    Within a script, it is very common to use the following pattern to get information about a specific job.
+
+    .. code-block:: sh
+
+        nnodes=$(flux jobs -no "{nnodes}" $FLUX_JOB_ID)
+
+    In order to get the number of nodes for the job we are running, we set the output format to exactly ``{nnodes}`` and nothing else.  The ``-n`` option ensures that the header from ``flux jobs`` will not be output.  So the only thing output from this call to ``flux jobs`` is just the number of nodes for the specified jobid, which we then store in the ``nnodes`` variable.
```

Comment:
Might be an unexpected indent here? Maybe I'm just seeing it weird on GitHub. Ignore this if it doesn't make a difference. 😅